spec/example_app/db/schema.rb

```diff
@@ -36,10 +36,9 @@
     t.string   "queue"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.index ["priority", "run_at"], name: "delayed_jobs_priority", using: :btree
```

Comment:
:smile: That was totally generated by Delayed Job/Rails, maybe this file should be ignored?